examples/multi-threading.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: UTF-8 -*-
+
+from fbchat import Client
+from fbchat.models import *
+import threading
+
+# Subclass fbchat.Client and override required methods
+class PrintMessage(Client):
```

Comment:
It seems wrong to call the client `PrintMessage`. Maybe something like `MessagePrinter` or `PrintBot` is better?

Reply:
`MessagePrinter` it is.